db-migrations/000005_adding_owners.up.sql

```diff
@@ -0,0 +1,22 @@
+-- schema changes for account owners and champions
+BEGIN;
+
+create table person(
+    id serial primary key,
+    login varchar not null,
+    email varchar not null,
+    name varchar not null,
+    valid boolean
+)
+
+create table champions(
+    foreign key(people_id) references person (id),
+    foreign key(aws_account_id) references aws_account (id)
+)
+
+create table owner(
+    foreign key(people_id) references person (id),
+    foreign key(aws_account_id) references aws_account (id)
+)
+
```

Comment:
task: add indexes, at least for the fields that will be queried frequently e.g. name, email